docs/getting_started/faq.rst

```diff
@@ -22,13 +22,28 @@ A: This means that either you provided the wrong password and/or the account
 you are trying to sign in with has 2FA enabled, and as such, either needs a 2FA
 token or a refresh token to sign in. A refresh token is preferred, because then
 you will not need to enter a 2FA token in order to sign in, and the session
-will last for longer than an hour. Refer to :ref:`2FA` and :ref:`code_flow` in
-order to use the respective auth methods.
+will last for longer than an hour. Refer to :ref:`2FA` and :ref:`refresh_token`
+in order to use the respective auth methods.
 
 .. _faq3:
 
 Q: Some options (like getting moderator logs from r/mod) keep on timing out.
 How can I extend the timeout?
 
 A: Set the timeout config option or initialize :class:`.Reddit` with a timeout
-of your choosing.
\ No newline at end of file
+of your choosing.
+
+.. _faq4:
+
+Q: Help, I keep on getting redirected to `/login`!
+
+A: PRAW is most likely in read-only mode. This normally occurs when you log in
+without a username and password or a refresh token. You need to sign in with either
+one of those. See :ref:`oauth_options` to see the avaliable methods
+to sign in. 
```

Comment:
Would something like this be better phrasing?

```suggestion
A: PRAW is most likely in read-only mode. This normally occurs when you authenticate
without a username and password or a refresh token. You need to authenticate with either
one of those. See :ref:`oauth_options` to see the available authentication methods.
```

Reply:
My previous comment was motivated by the ambiguity of "you need to" — *why* do I need to, or to achieve *what*? So I'd like something explaining the purpose of signing in.

Reply:
In that case, would this be better:
```suggestion
A: PRAW is most likely in read-only mode. This normally occurs when PRAW is
authenticated without a username and password or a refresh token. In order to perform
this action, the Reddit instance needs to be authenticated. See :ref:`oauth_options` to
see the available authentication methods.
```

Reply:
Yes, that's great.